src/server/user.ts

```diff
@@ -32,43 +30,13 @@ export async function setFavoriteAd(adId: number): Promise<boolean> {
     return answer;
 };
 
-export async function fetchRamadAds(): Promise<RamadAds[]> {
-    // TODO: insert DDL
-    const ramadAds: RamadAds[] = [
-        {
-            id: 1,
-            name: 'דרוש רשץ דחוף',
-            role: {id:1, name: 'רש"צ', color: 'red', initials: 'TL'},
-            candidates: [ {
-                name: 'אדיר סיטון',
-                phoneNumber: '052-4231561',
-                upn: 's8232392' 
-            }, {
-                name: 'מיכאל הופמן',
-                phoneNumber: '052-4243361',
-                upn: 's8189435'                 
-            } ],
-            isClosed: true
-        }, {
-            id: 2,
-            name: 'מנהל מוצר תותח',
-            role: {id:1, name: 'מנהל מוצר', color: 'red', initials: 'TL'},
-            candidates: [ {
-                name: 'אדיר סיטון',
-                phoneNumber: '052-4231561',
-                upn: 's8232392' 
-            }, {
-                name: 'מיכאל הופמן',
-                phoneNumber: '052-4243361',
-                upn: 's8189435'                 
-            }, {
-                name: 'אאאאאבבבבבגגגגגדדדדד הההההווווו',
-                phoneNumber: '054-3592234',
-                upn: 's7464334'                 
-            } ],
-            isClosed: false
-        }
-    ]
+export async function fetchRamadAds(): Promise<RamadAd[]> {
+    const ramadAdsSQL: RamadAdsSQL[] = await fetch('/user/ramad-ads', {
+        method: 'GET'
+    }).then(response => {
```

Comment:
Instead of .then, maybe .json() on the fetch. Since this is await, it might wait 🤔

Reply:
Excellent worked, Done

Reply:
actually I prefer that way, because it came out a little bit weird
    `const favoriteAds: number[] = await (await fetch(`/user/favorite`)).json()`
await * 2

Reply:
so I leave it like that